jina/drivers/__init__.py

```diff
@@ -215,21 +219,21 @@ def __getstate__(self) -> Dict[str, Any]:
 
 class BaseRecursiveDriver(BaseDriver):
 
-    def __init__(self, recur_range: Tuple[int] = (0, 1), apply_order: str = 'post',
-                 recur_on: Tuple[str] = ('chunks',), *args, **kwargs):
+    def __init__(self, recur_depth_range: Tuple[int, int] = (0, 1), recur_adjacency_range: Tuple[int, int] = (0, 0),
```

Comment:
This I put to keep exactly the default `chunks` recur_on (I translate `recur_on` = 'chunks'' to `recur_depth_range: Tuple[int, int] = (0, 1)`. Not that I have a strong opinion on that, but to keep the changes at minimmum

Reply:
Seeing how often you did `recur_depth_range: [0, 0]`, I still vote for `(0,0)` as the default. ;) Anyhow, since this PR is for getting us running for the Hackaton, keep it with `(0,1)` and we might refactor this (or not) in the big change to come when we restructure the rankers.